setup.py

```diff
@@ -13,10 +14,14 @@
     'zope.app.container',
 ]
 
-try:
-    import json
-except ImportError:
-    requirements.append('simplejson')
+if version_info[0] == 2 and version_info[1] < 6:
+    # If this happens you are probably using Python 2.4
+    # According to the documentation of simplejson, since version 2.1.0,
+    # Python 2.4 is no longer supported but may still work
+    # This line will probably pull in simplejson 2.3.3
+    requirements.append('simplejson < 2.4')
+    # To my knowledge 1.3 work fine
+    requirements.append('plone.app.transmogrifier < 1.4')
```

Comment:
I found it surprisingly working!
```
(testenv2.4)ale@emily:~/Code/testenv2.4/collective.jsonmigrator$ ../bin/python2.4 setup.py install 2> /dev/null |grep plone.app.transmogrifier
Searching for plone.app.transmogrifier<1.4
Reading http://pypi.python.org/simple/plone.app.transmogrifier/
Best match: plone.app.transmogrifier 1.3
```
But I see couple of glitches now:

1. there are some `except Exception as e` which are not python2.4 compatible
2. the produced `requires.txt` is not aware of this version limitation

I will try to fix both.